src/libraries/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs

```diff
@@ -69,6 +69,159 @@ public async Task ServerAsyncAuthenticate_EachSupportedProtocol_Success(SslProto
             await ServerAsyncSslHelper(SslProtocolSupport.SupportedSslProtocols, serverProtocol);
         }
 
+        [Fact]
+        public async Task ServerAsyncAuthenticate_SimpleSniOptions_Success()
+        {
+            var state = new Object();
+            var serverOptions = new SslServerAuthenticationOptions() { ServerCertificate = _serverCertificate };
+            var clientOptions = new SslClientAuthenticationOptions() { TargetHost = _serverCertificate.GetNameInfo(X509NameType.SimpleName, false) };
+            clientOptions.RemoteCertificateValidationCallback = (sender, certificate, chain, sslPolicyErrors) => true;
```

Comment:
I don't really care about this but for future reference if you aren't using any of the parameters you can also do:
```suggestion
            clientOptions.RemoteCertificateValidationCallback = delegate { return true; }
```